app/models/conversation.rb

```diff
@@ -83,7 +83,7 @@ def ordered_participants
   end
 
   def subject
-    self[:subject].blank? ? "no subject" : self[:subject]
+    self[:subject].blank? ? I18n.t('conversations.new.subject_default') : self[:subject]
```

Comment:
I used a single quote as that seemed the most common. Do you mean like this?

> I18n.t("conversations.new.subject_default")

?